docs/reference_pickerfield.md

```diff
@@ -0,0 +1,142 @@
+---
+id: reference_pickerfield
+title: <picker-field>
+sidebar_label: <picker-field>
+---
+
+The `<picker-field>` element represents a single-line input field. When pressed, the field focused and a modal appears, allowing the user to select one of the available options. The modal uses a platform-specific widget:
+
+- on Android, it shows the system picker modal
+- on iOS, the spinner-picker appears at the bottom of the screen
+
+A picker field with no value selected:
+![unselected](/img/reference_pickerfield_unselected.png)
+
+On iOS, pressing the field brings up a bottom sheet modal with a picker
+![focused](/img/reference_pickerfield_focused.png)
+
+After selecting a value and pressing "Done", the field has the value selected.
+![focused](/img/reference_pickerfield_selected.png)
+
+Like other input elements, the selected value of the picker gets serialized by the parent form when making requests to the server.
+
+```xml
+<picker-field field-style="Input"
```

Comment:
Should we also add the `name` attribute here? Since `name` would be required almost every time when working with a form?